proxied.d.ts

```diff
@@ -669,7 +670,9 @@ export interface Typegram<F> {
       file_id: string;
     }): File;
 
-    /** Use this method to kick a user from a group, a supergroup or a channel. In the case of supergroups and channels, the user will not be able to return to the chat on their own using invite links, etc., unless unbanned first. The bot must be an administrator in the chat for this to work and must have the appropriate admin rights. Returns True on success. */
+    /**
+     * @deprecated
```

Comment:
Maybe highlight what to use instead?
```suggestion
     * @deprecated use banChatMember instead
```